app/src/main/java/app/grapheneos/camera/CamConfig.kt

```diff
@@ -532,6 +531,10 @@ class CamConfig(private val mActivity: MainActivity) {
         camera!!.cameraInfo.isZslSupported
     }
 
+    val isStabilizationSupported : Boolean by lazy {
+        Recorder.getVideoCapabilities(camera!!.cameraInfo).isStabilizationSupported
```

Comment:
Is `camera` guaranteed to be non-null at this point?

Reply:
Thanks a lot for pointing this out @muhomorr!

While this is guaranteed to be true for the normal `CaptureActivity`, it wouldn't have been true when the video mode has been directly invoked for example in case of `android.media.action.VIDEO_CAMERA` intent or similar intents supported by our app.

To fix this issue, we could instead rely on getting the list of `CameraInfo` objects from the `cameraProvider` object present in the `CamConfig` class that is guaranteed to be non-null when `startCamera` method is called and then select the `CameraInfo` object based on its `lensFacing` which would be guaranteed to be available otherwise  since we ensure that before calling `startCamera` in our app.

The `CameraInfo` object that would then be retrieved using the above method could be used to be passed instead to make this robust regardless of the use cases being bind or not.

I have pushed the changes related to the same to the branch. Please do let me know if any further changes are needed. Thanks for your valuable time and help!